modules/apps/headless/headless-object/headless-object-impl/src/main/java/com/liferay/headless/object/internal/resource/v1_0/ObjectEntryFolderResourceImpl.java

```diff
@@ -303,6 +307,28 @@ private ObjectEntryFolder _addObjectEntryFolder(
 				).build()));
 	}
 
+	private long _getGroupId(String scopeKey) throws Exception {
+		try {
+			Group group = _groupService.getGroup(
+				contextCompany.getCompanyId(), scopeKey);
+
+			if (group != null) {
+				return group.getGroupId();
+			}
+		}
+		catch (PortalException portalException) {
+			if (_log.isDebugEnabled()) {
+				_log.debug(portalException);
+			}
+		}
+
+		if (!Validator.isNumber(scopeKey)) {
+			throw new NoSuchGroupException();
+		}
+
+		return GetterUtil.getLong(scopeKey);
+	}
+
```

Comment:
Cannot we use `com.liferay.portal.vulcan.util.GroupUtil.getGroupId()` here?

Reply:
@adolfopa, we allow scopeKey to be 0, so we cannot use GroupUtil. But I will change so as to not use try/catch by using localService